provider/resources.go

```diff
@@ -256,12 +256,17 @@ func Provider() tfbridge.ProviderInfo {
 							TypeSpec:    schema.TypeSpec{Type: "string"},
 						},
 						"repoDigest": {
-							Description: "The manifest digest of an image pushed to a registry, of the format " +
+							Description: "**For pushed images:**\n" +
+								"The manifest digest of an image pushed to a registry, of the format " +
 								"repository@<algorithm>:<hash>, e.g. `username/demo-image@sha256:" +
 								"a6ae6dd8d39c5bb02320e41abf00cd4cb35905fec540e37d306c878be8d38bd3`.\n" +
 								"This reference is unique per image build and push. \n" +
 								"Only available for images pushed to a registry.\n" +
-								"Use when passing a reference to a pushed image to container management resources.",
+								"Use when passing a reference to a pushed image to container management resources.\n\n" +
+								"**Local-only images**" +
+								"For local images, this field is the image ID of the built local image, of the format " +
+								"<algorithm>:<hash>, " +
```

Comment:
This is all probably fine as is - but another way to frame this would be:
* The format is always `[<repository>@]<algorithm>:<hash>`
* The `<repository>@` component is only available for images pushed to a remote registry, and is the qualified repository name in that registry.
* The `<algorithm>:<hash>` comoponent will be the image digest from the registry if the image was pushed, or else the ImageId of the image in the local Docker system if the image was not pushed.
* In both cases, this can be passed to `docker run` or to `docker.Container` as a unique name for this build of the container image.